### DISCUSSION TITLE
Fork Semantic-UI-React into this org and let people contribute

### DISCUSSION THREAD

**Discussion by kevinresol** [fomantic/Fomantic-UI#1771] [General] [answered]
As it is explicitly stated by the SUI-React maintainer that it won't support fomantic. So IMO fomantic as a community-supported org should just fork it and make it a starting point for people to contribute.

Also, FUI 3.0 is nowhere near completion I think we should just move forward with a forked SUI react for FUI 2.x.

**Comment by lubber-de** [comment]
AFAIK the SUI react people are working on a provider layer to support fomantic.
I am turning this issue into a discussions thread

**Comment by lubber-de** [comment, marked as answer]
We would need volunteering, responsible active maintainers with internal SUIR knowledge for such a task. Even if people would contribute, there must be some people doing serious reviews for quality merging, understanding the existing SUIR Core library in it's detail to make sure everything works altogether. Those people should also be aware for additional active development whenever FUI is updated.

The FUI core team can't accomplish this. So "just" forking SUIR is the wrong sign to the community IMHO. They will at least expect an initial release to be fully compatible to FUI (not even SUIR is fully compatible to SUI )

If we just fork it, most people will start complaining that several things won't work (because they simply don't exist in SUIR yet like calendar, slider, toast). And with nobody actively maintaining it, it's likely to become the same situation as SUI is today.

Remember, at least all (missing) JS modules have to be recreated in a FUIR which is quite a challenging task.

@kevinresol Would you like to become a possible first active maintainer of such a fork? 😉
